packages/react/ssr-tests/nextjs/pages/_app.js

```diff
@@ -17,16 +21,42 @@ const langCode = {
  * Class IbmdotcomLibrary
  */
 export default class IbmdotcomLibrary extends App {
+  /**
+   * Loads in the initial query string parameters
+   *
+   * @param {object} props page props
+   * @param {object} props.ctx app context
+   * @param {object} props.query Query params object
+   * @returns {Promise<{pageProps}>} Returns the pageProps
+   */
+  static async getInitialProps({ ctx, query }) {
+    let host;
+
+    if (ctx.req) {
+      host = ctx.req.headers.host;
+    } else {
+      host = location.hostname;
+    }
+
+    const useLang =
+      query && query.lc ? { lc: query.lc, cc: query.cc } : _defaultLang;
+
+    return { useLang, host };
+  }
+
   /**
    * Renders the DotcomShell
    *
    * @returns {*} Page wrapper JSX
    */
   render() {
-    const { Component, pageProps } = this.props;
+    const { Component, useLang, host, pageProps } = this.props;
     return (
       <Container>
-        <DotcomShell navigation="default" langCode={langCode}>
+        <Head>
+          <Altlang host={host} lang={useLang} />
```

Comment:
we don't need the `lang` prop here right? Don't see it being used in `<Altlang />`

Reply:
thanks, good catch! pushing a fix.